data-raw/process_data_sets.R

```diff
@@ -272,3 +272,29 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+# Electric vehicle charging sessions
+# information from 3,395 high resolution electric vehicle charging sessions
+# Original data from: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/NFPQLW
```

Comment:
If you are reading in data from an external URL, you don't need `data-raw/station_data_dataverse.csv`.